src/stories/components/Steps/Steps.js

```diff
@@ -0,0 +1,73 @@
+import './styles.scss';
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * Component for step prgress element.
+ *
+ * @component
+ * @param {number} props of the component.
+ * @param {array} labels of each step.
+ * @return {object} (
+ *   <Steps props={props
+ *      labels={labels} />
+ * )
```

Comment:
Just curious if the `number` parameter is kind of a generic param that can take the form of any of `totalSteps`, `currentStep`, and/or `goToStep`. I saw in the Steps.stories.js file you give all of those 3 as props of `<Steps/>`, so I was wondering if we could simplify the documentation and only have one @param, which would be called "props".

Maybe I am just overcomplicating things so just disregard this comment if that's the case.

Reply:
I honestly wasn't sure if the param can be `prop` or it should be object or whatnot; I'll do some googling!